src/HttpClientToCurl/Generator.cs

```diff
@@ -1,35 +1,36 @@
 using System.Data;
 using System.Text;
+using HttpClientToCurl.Config;
 
 namespace HttpClientToCurl;
 
 public static class Generator
 {
-    public static string GenerateCurl(HttpClient httpClient, HttpRequestMessage httpRequestMessage, bool needAddDefaultHeaders)
+    public static string CreateCurl(HttpClient httpClient, HttpRequestMessage httpRequestMessage, BaseConfig config)
```

Comment:
@naeemaei this class and all methods have been called inside the public methods that are used by developers.
There are all of the public methods in the Main class.
also, this method is public because I want to write unit test scenarios for this method.

Reply:
Well done, thanks!